.github/workflows/image_build_push.yaml

```diff
@@ -0,0 +1,13 @@
+name: Build Image and Push to Quay
```

Comment:
I think `uc-cdis/.github/.github/workflows/image_build_push.yaml` pushes to ECR by default as well. Just a thought, but maybe this could be `Build Image and Push`, `Build Image and Push to Registries`, or `Build Image and Push to Quay and ECR`.

Reply:
Nice! I like the middle one (Registries) - more generic and abstract.